perf/local_lat.js

```diff
@@ -19,7 +19,7 @@ rc = s.bind(bind_to);
 assert(rc >= 0);
 
 var i = 0;
-s.on('message', function (data) {
+s.on('data', function (data) {
```

Comment:
well yes, but i was thinking about activating the new API with an initializing flag option given to the socket since there's a performance hit involved in switching to the streams API